dateparser_data/settings.py

```diff
@@ -21,4 +21,5 @@
     'RETURN_TIME_AS_PERIOD': False,
     'PARSERS': default_parsers,
     'REQUIRE_PARTS': [],
+    'USE_GIVEN_LANGUAGE_ORDER': False
```

Comment:
I think `False` is right as a default because that way you'd get the added benefit of trying your most likely language first. If you give a list of locales I'd assume you want the date to be extracted correctly *regardless* of which of your locales is the right one. Also, I would argue that expecting it to be as efficient as possible by default is better if you allow the customization.